clients/drcachesim/common/options.cpp

```diff
@@ -982,6 +982,13 @@ droption_t<bool> op_encodings2regdeps(
     "This option is for -simulator_type " RECORD_FILTER ". When present, it converts "
     "the encoding of instructions from a real ISA to the DR_ISA_REGDEPS synthetic ISA.");
 
+droption_t<std::string>
+    op_filter_func_markers_by_value(DROPTION_SCOPE_FRONTEND,
```

Comment:
See earlier comment: "by_value" sounds like the value of markers such as _ARG or _RETVAL.  I think this should be `-filter_func_markers_by_id` or even better just `-filter_func_ids`.

Reply:
Went with "-filter_keep_func_ids" as discussed.